src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
I tested the example in Chrome for Android, and it worked like charm...

I think the feature detection I implemented should be pretty rock solid, seeing as the tests didn't break and Chrome for Android works... however you say Safari broke, but I'm on Linux so  I can't test it. Might ask a friend with a mac.